### PR TITLE
(ASC-1365) Execute swift client on util container

### PR DESCRIPTION
**Pull request by phongdly**
Prior to this PR, test_swift_stat.py is skipped (xFail) on Rocky
onward.

This PR to allow the test run on Rocky.

### CODE REVIEW

File: molecule/default/tests/test_dhcp_agents.py [hunk 1]
```diff
@@ -17,9 +18,8 @@ def test_openvswitch(host):
     Ensure DHCP agents for all networks are up
     """
 
-    r = \
-        (host.ansible("setup")["ansible_facts"]["ansible_local"]
-            ["system_tests"]["rpc_product_release"])
+    r = next(tmp_var.gen_dict_extract('rpc_product_release',
```
Comment (johnduarte) [inline molecule/default/tests/test_dhcp_agents.py]: This refactor of how the `rpc_product_release` does not appear to be directly related to the swift issue. Can you break these changes out into a separate PR?

File: molecule/default/tests/test_swift_stat.py [hunk 2]
```diff
@@ -11,22 +12,26 @@
...
-    if not major.isdigit() or major > 17:
-        pytest.xfail("openrc credentials are not available "
-                     "on the swift proxy node past queens")
-
-    result = helpers.run_on_swift('swift stat -v', host)
+    cmd1 = 'swift stat -v'
+    if major < 17:
+        result = helpers.run_on_swift(cmd1, host)
+    else:
+        cmd2 = ("{} {} {}".format(os_pre, cmd1, os_post))
```
Comment (johnduarte) [inline molecule/default/tests/test_swift_stat.py]: This section can use the [run_on_container](https://github.com/rcbops/pytest-rpc/blob/master/pytest_rpc/helpers.py#L547) helper instead.

File: molecule/default/tests/test_swift_stat.py [hunk 3]
```diff
@@ -11,22 +12,26 @@
 testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
     os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('shared-infra_hosts')[:1]
 
+os_pre = ("lxc-attach -n $(lxc-ls -1 | grep utility | head -n 1) "
+          "-- bash -c '. /root/openrc ; ")
+os_post = "'"
```
Comment (johnduarte) [inline molecule/default/tests/test_swift_stat.py]: This code is not necessary if the [run_on_container](https://github.com/rcbops/pytest-rpc/blob/master/pytest_rpc/helpers.py#L547) helper is used below.

**Comment by phongdly** [comment]
This PR is blocked until PR#159 is merged
